docs/examples/zoom-factory.ipynb

```diff
@@ -69,7 +69,7 @@
     "## Scrolling and panning\n",
     "\n",
     "\n",
-    "I like being able to pan by clicking with the mouse without having to click the toolbar. Below is an example demonstrating the {class}`.panhandler` object. It allows scrolling using the middle mouse button, but with the `button` argument, you can change that to any mouse button. You need to make sure to assign the panhanler to a variable otherwise it will be garbage collected and will not work."
+    "I like being able to pan by clicking with the mouse without having to click the toolbar. Below is an example demonstrating the {class}`.panhandler` object. It allows for scrolling/panning using the middle/right mouse buttons, but with the `button` argument, you can change that to any mouse button. You need to make sure to assign the panhanler to a variable otherwise it will be garbage collected and will not work."
```

Comment:
I don't know what I was thinking with "scrolling" it really should just be "panning". 

Also the default is actually just the right mouse button. It used to be that middle and right click were the same in jupyter backends so you had to listen to right click to detect middle clicks. But that got fixed here: https://github.com/matplotlib/matplotlib/pull/18172

Maybe the default should be changed - default right click is kinda awkward, what do you think?

Either way now that middle and right are differentiable lets at least correctly state that it is only the "right mouse button"

Reply:
```suggestion
    "I like being able to pan by clicking with the mouse without having to click the toolbar. Below is an example demonstrating the {class}`.panhandler` object. It allows for panning by clicking and dragging even if the toolbar pan widget is not active. You can change which button is used via the `button` argument. Make sure to assign the panhandler to a variable otherwise it will be garbage collected and will not work."
```